gen_pools_and_gauges.py

```diff
@@ -70,9 +70,45 @@ def process_query_preferential_gauges(result) -> dict:
     return df.set_index("symbol")["id"].to_dict()
 
 
+def process_query_root_gauges(result, gauges) -> dict:
```

Comment:
Does it make sense to move this kind of logic to the subgraphs object?

Reply:
not really, this is very specific to the `gen_pools_and_gauges.py` workflow imo